docs/04-resources/04-tutorials/rootstock-wagmi-starter-kit.md

```diff
@@ -0,0 +1,316 @@
+  - `useContract:` A hook to load and interact with a specific smart contract.
+  - `useAccount:` A hook to manage the user’s Ethereum account.
+  - `useBalance:` A hook to retrieve the account balance.
+  - `useContractEvents:` A hook to listen to contract events.
+
+5. **Ethereum Development Made Easier:** Whether you’re building decentralized applications (DApps), DeFi projects, or NFT platforms, Wagmi aims to simplify the development process by abstracting away low-level details and providing a more intuitive API.
+
+## What is Shadcn?
+Shadcn is a library that extends styling capabilities beyond what’s offered by standard CSS frameworks. It allows developers to create unique and visually appealing designs by adding shadows, gradients, and other effects to their UI elements.
+
+## Prerequisites
+- **Node.js and Git Installation:** Make sure you have Node.js and Git installed on your system. If not, download and install them from their official websites.
+- **Yarn Installation:** Install Yarn, a package manager for Node.js projects. You can do this by running the following command in your terminal:
+  ```bash
+  npm install -g yarn
+  ```
+- **Basic Knowledge:** Familiarize yourself with React (a JavaScript library for building user interfaces) and Solidity (a programming language for Ethereum smart contracts).
+
+## Setup
+
+### 1. Clone the Repository
+First, you’ll need to clone the Rootstock Wagmi Starter Kit repository. Open your terminal and run the following commands:
+```bash
+git clone https://github.com/chrisarevalodev/rsk-wagmi-starter-kit.git
+cd rsk-wagmi-starter-kit
+```
+
+### 2. Get Project ID
+Every dApp that relies on WalletConnect now needs to obtain a projectId from [WalletConnect Cloud](https://cloud.walletconnect.com/). This is free and only takes a few minutes.
+
+To get the key:
+1. Go to this [URL](https://cloud.walletconnect.com/sign-up) and sign up.
+2. Create a new project by clicking on **Create Project**.
+3. Add a Name and Link to your project, select a product (AppKit or WalletKit), select **WalletKit**.
+4. Now you will see your project ID, copy it.
+
+### 3. Environment Setup
+To set up your environment, follow these steps:
+1. Create a `.env` file and add your environment variables.
+    ```env
+    VITE_WC_PROJECT_ID=Your projectid from cloud Walletconnect
+    ```
+
+### 4. Install Dependencies
+Before running the project, make sure you have the necessary dependencies installed. We recommend using the yarn package manager due to potential conflicts with npm packages. Run the following command to install dependencies:
+```bash
+yarn
+```
+
+### 5. Run the Project
+Now that you’ve cloned the repository and installed dependencies, it’s time to run the project. Execute the following command:
+```bash
+yarn dev
+```
+This will start your Rootstock Wagmi Starter Kit locally, allowing you to develop and test your smart contracts. You can access the Vite server at [http://localhost:5173](http://localhost:5173).
```

Comment:
```suggestion
This will start the Rootstock Wagmi Starter dApp locally, allowing you to develop and test your smart contracts. You can access the Vite server at [http://localhost:5173](http://localhost:5173).
```